assemblies/assembly-configuring-authorization-in-rhdh.adoc

```diff
@@ -0,0 +1,43 @@
+[id='configuring-authorization-in-rhdh']
+= Configuring authorization in {product}
+
+include::modules/admin/con-rbac-overview.adoc[leveloffset=+1]
+
+
+include::modules/admin/ref-rbac-permission-policies.adoc[leveloffset=+1]
```

Comment:
@themr0c Permission policies are a component of the RBAC feature. Currently, it seems that permission policies and RBAC are presented as two distinct options for configuring authorization. This hierarchy appears to be suitable for now, given that we only have the RBAC feature. However, as we introduce additional authorization options in the future, we may need to reconsider and revise this structure. what do you think?

Reply:
Agree. It was out of scope of this PR.